draft-packages/form/KaizenDraft/Form/TextAreaField/README.mdx

```diff
@@ -0,0 +1,151 @@
+
+<DoAndDontContainer>
+<Do>
+
+- e.g. Thanks for everything!
+
+</Do>
+
+<Dont>
+
+- Thanks for everything!
+
+</Dont>
+</DoAndDontContainer>
+
+Keep in mind that examples aren’t always required. For example, a text area asking for someone’s opinion doesn’t need an example in it if it will be misleading. Only use placeholder text when required. More often than not, blank text fields are easier to spot and scan.
+
+### Error text should be succinct
+
+Keep all error text short, useful and to the point.
+
+<DoAndDontContainer>
+<Do>
+
+- You must enter a comment
+
+</Do>
+
+<Dont>
+
+- You entered a comment that is invalid
+
+</Dont>
+</DoAndDontContainer>
+
+### Helper text should provide extra guidance
+
+Helper text placed below the field can include further instructions or information on how to fill out the text area. It can also include details on what will happen with the filled out information. Don’t put placeholders here.
+
+<DoAndDontContainer>
+<Do>
+
+- Your comment will be used anonymously
+
+</Do>
+
+<Dont>
+
+- For example, “Thanks for everything!”
+
+</Dont>
+</DoAndDontContainer>
+
+### Write in sentence case with minimal punctuation
+
+All text area copy should be in sentence case i.e. the first letter of the first word is capitalized and everything else is in lower case unless it’s a proper noun or feature name. Always err on the side of minimal punctuation, so leave out full stops and commas where possible.
+
+## To keep in mind
+
+* **Optional vs required:**
+    * As a rule of thumb, generally use “ (optional)” to indicate optional fields instead of “`*`” to indicate required fields.
+* **Placeholder text** is temporary and vanishes once a user enters text. Avoid placeholder text that must be deleted by the user. Avoid using placeholder text as a label as the user will lose the context of what field they're filling out as soon as it disappears.
+* For **sizing**, consider likely text input and use existing data where possible.
+* For **right-to-left (RTL)** languages, the layout of a text area is swapped. The validation icons are left-aligned.
+
+### Validation
+
+Most of the time, we validate text areas on “blur” (when leaving the text area) or on form submission as needed. We don’t validate on key strokes.
+
+When displaying a validation messages, such as an error, use a `aria-describedby` attribute to associate the text field with the element containing the validation message.
+
+### Interaction
+
+See [Interaction states guidelines for disabled elements](/guidelines/interaction-states#disabled-state).
+
+## When to use and when not to use
+
+<WhenToUseAndWhenNotToUse>
+<WhenToUse>
+
+* Use a Text Area Field to gather longer text input.
+
+</WhenToUse>
+<WhenNotToUse>
+
+* Avoid gathering input from users that isn't absolutely necessary.
+* For shorter text input, use a [Text Field](/components/text-field) instead.
```

Comment:
How does this component work with the Rich Text editor? Should we also include guidelines as to when to use this vs. Rich Text editor, or is this actually considered part of the RTE?